<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Evolutionary Transcriptomics • myTAI</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="Evolutionary Transcriptomics">
<meta property="og:description" content="Investigate the evolution of biological processes by capturing evolutionary signatures in transcriptomes (Drost et al. (2018) &lt;doi:10.1093/bioinformatics/btx835&gt;). This package aims to provide a transcriptome analysis environment to quantify the average evolutionary age of genes contributing to a transcriptome of interest. ">
<meta property="og:image" content="/logo.png">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">myTAI</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.0.1.9000</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="articles/Advanced.html">Advanced Phylotranscriptomics Analyses</a>
    </li>
    <li>
      <a href="articles/Enrichment.html">Enrichment Analyses</a>
    </li>
    <li>
      <a href="articles/Expression.html">Gene Expression Analysis with `myTAI`</a>
    </li>
    <li>
      <a href="articles/Intermediate.html">Intermediate Phylotranscriptomics Analyses</a>
    </li>
    <li>
      <a href="articles/Introduction.html">Introduction to the myTAI package</a>
    </li>
    <li>
      <a href="articles/Phylostratigraphy.html">Discussing Genomic Phylostratigraphy</a>
    </li>
    <li>
      <a href="articles/Taxonomy.html">Retrieving Taxonomic Information</a>
    </li>
  </ul>
</li>
<li>
  <a href="news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/drostlab/myTAI/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">

<div class="section level1">
<div class="page-header"><h1 id="mytai-">myTAI <img src="reference/figures/logo.png" align="right" height="174" width="150"><a class="anchor" aria-label="anchor" href="#mytai-"></a>
</h1></div>
<p><a href="https://github.com/r-hub/cranlogs.app" class="external-link"><img src="http://cranlogs.r-pkg.org/badges/myTAI" alt="rstudio mirror downloads"></a> <a href="https://github.com/r-hub/cranlogs.app" class="external-link"><img src="http://cranlogs.r-pkg.org/badges/grand-total/myTAI" alt="rstudio mirror downloads"></a> <a href="https://academic.oup.com/bioinformatics/article/34/9/1589/4772684" class="external-link"><img src="https://img.shields.io/badge/Published%20in-Bioinformatics-126888.svg" alt="Paper link"></a> <a href="https://anaconda.org/bioconda/r-mytai" class="external-link"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat" alt="install with bioconda"></a> <img src="https://api.visitorbadge.io/api/visitors?path=https%3A%2F%2Fgithub.com%2Fdrostlab%2FmyTAI&amp;label=VISITORS&amp;countColor=%23263759&amp;style=flat" alt="Visitors"></p>
</div>
<div class="section level1">
<h1 id="evolutionary-transcriptomics-with-r">Evolutionary Transcriptomics with R<a class="anchor" aria-label="anchor" href="#evolutionary-transcriptomics-with-r"></a>
</h1>
<div class="section level3">
<h3 id="motivation">Motivation<a class="anchor" aria-label="anchor" href="#motivation"></a>
</h3>
<p>In the past years, a variety of studies aimed to uncover the molecular basis of morphological innovation and variation from the evolutionary developmental perspective. These studies often rely on transcriptomic data to establish the molecular patterns driving the complex biological processes underlying phenotypic plasticity.</p>
<p>Although transcriptome information is a useful start to study the molecular mechanisms underlying a biological process of interest (molecular phenotype), they rarely capture <strong>how</strong> these expression patterns emerged in the first place or to <strong>what extent</strong> they are possibly constrained, thereby neglecting the evolutionary history and developmental constraints of genes contributing to the overall pool of expressed transcripts.</p>
<p>To overcome this limitation, the <code>myTAI</code> package introduces procedures summarized under the term <code>evolutionary transcriptomics</code> to integrate gene age information into classical gene expression analysis. <code>Gene age inference</code> can be performed with <a href="">various existing software</a>, but we recommend using <a href="https://github.com/josuebarrera/GenEra" class="external-link">GenEra</a> or <a href="https://github.com/kullrich/orthomap" class="external-link">orthomap</a>, since they address published shortcomings of <code>gene age inference</code> (<a href="https://drostlab.github.io/myTAI/articles/Phylostratigraphy.html" class="external-link">see detailed discussion here</a>). In addition, users can easily retrieve <a href="https://github.com/HajkD/published_phylomaps" class="external-link">previously precomputed gene age information</a> via our data package <a href="https://github.com/LotharukpongJS/phylomapr" class="external-link">phylomapr</a>.</p>
<p>Evolutionary transcriptomics studies can serve as a first approach to screen <em>in silico</em> for the potential existence of evolutionary and developmental constraints within a biological process of interest. This is achieved by quantifying transcriptome conservation patterns and their underlying gene sets in biological processes. The exploratory analysis functions implemented in <code>myTAI</code> provide users with a standardized, automated and statistically sound framework to detect and analyze patterns of evolutionary constraints in any transcriptome dataset of interest.</p>
</div>
<div class="section level3">
<h3 id="detailed-documentation-provided-here">Detailed documentation provided <a href="https://drostlab.github.io/myTAI/articles/Introduction.html" class="external-link">here</a>
<a class="anchor" aria-label="anchor" href="#detailed-documentation-provided-here"></a>
</h3>
</div>
<div class="section level2">
<h2 id="citation">Citation<a class="anchor" aria-label="anchor" href="#citation"></a>
</h2>
<p>Please cite the following paper when using <code>myTAI</code> for your own research. This will allow me to continue working on this software tool and will motivate me to extend its functionality and usability in the next years. Many thanks in advance!</p>
<blockquote>
<p>Drost et al. <strong>myTAI: evolutionary transcriptomics with R</strong>. <em>Bioinformatics</em> 2018, 34 (9), 1589-1590. <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btx835/4772684" class="external-link">doi:10.1093</a></p>
</blockquote>
<div class="section level3">
<h3 id="dependencies">Dependencies<a class="anchor" aria-label="anchor" href="#dependencies"></a>
</h3>
<p><code>myTAI</code> is using <code>OpenMP</code> to run cpp code in parallel, thus you might need to install additional software.</p>
<p><strong>Mac:</strong></p>
<div class="sourceCode" id="cb1"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="ex">brew</span> install llvm libomp</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> /usr/local/lib</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ln</span> <span class="at">-s</span> /usr/local/opt/libomp/lib/libomp.dylib ./libomp.dylib</span></code></pre></div>
<p><strong>Linux:</strong> 1. Check if you have an OpenMP-enabled compiler: Open a terminal and run the command <code>gcc --version</code> or <code>g++ --version</code> to check if you have the GNU Compiler Collection (GCC) installed. OpenMP support is typically included in GCC.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>If you don<span class="st">'t have GCC installed, you can install it using your distribution'</span>s package manager. For example, on Ubuntu, you can use sudo apt<span class="sc">-</span>get install build<span class="sc">-</span>essential to install GCC.</span></code></pre></div>
<ol start="2" style="list-style-type: decimal">
<li>
<p>Install the libomp library: Open a terminal and run the command appropriate for your package manager:</p>
<p>On Ubuntu or Debian-based systems: <code>sudo apt-get install libomp-dev</code> or <code>sudo apt install libomp-dev</code></p>
<p>On Fedora or CentOS systems: <code>sudo dnf install libomp-devel</code></p>
<p>On Arch Linux: <code>sudo pacman -S libomp</code></p>
</li>
</ol>
<p><strong>Windows:</strong></p>
<p>On Windows, the most common OpenMP implementation is provided by Microsoft Visual C++ (MSVC) compiler, which includes OpenMP support by default. So, if you are using MSVC as your C++ compiler, you should have OpenMP support without any additional installations.</p>
</div>
<div class="section level3">
<h3 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h3>
<p>Users can install <code>myTAI</code> from <a href="https://cran.r-project.org/web/packages/myTAI/index.html" class="external-link">CRAN</a>:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># install myTAI 0.9.3</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"myTAI"</span>, dependencies <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="short-package-description">Short package description<a class="anchor" aria-label="anchor" href="#short-package-description"></a>
</h3>
<p>Using <code>myTAI</code>, any existing or newly generated transcriptome dataset can be combined with evolutionary information (find <a href="https://drostlab.github.io/myTAI/articles/Introduction.html#retrieval-of-phylogenetic-or-taxonomic-information" class="external-link">details here</a>) to retrieve novel insights about the evolutionary conservation of the transcriptome at hand.</p>
<p>For the purpose of performing large scale evolutionary transcriptomics studies, the <code>myTAI</code> package implements the quantification, statistical assessment, and analytics functionality to allow researchers to study the evolution of biological processes by determining stages or periods of evolutionary conservation or variability in transcriptome data.</p>
<p>We hope that <code>myTAI</code> will become the community standard tool to perform evolutionary transcriptomics studies and we are happy to add required functionality upon request.</p>
</div>
<div class="section level3">
<h3 id="scientific-background">Scientific background<a class="anchor" aria-label="anchor" href="#scientific-background"></a>
</h3>
<p>Today, phenomena such as morphological mutations, diseases or developmental processes are primarily investigated on the molecular level using transcriptomics approaches. Transcriptomes denote the total number of quantifiable transcripts present at a specific stage in a biological process. In disease or developmental (defect) studies, transcriptomes are usually measured over several time points. In treatment studies aiming to quantify differences in the transcriptome due to biotic stimuli, abiotic stimuli, or diseases usually treatment / disease versus non-treatment / non-disease transcriptomes are compared. In either case, comparing changes in transcriptomes over time or between treatments allows us to identify genes and gene regulatory mechanisms that might be involved in governing the biological process of investigation. Although classic transcriptomics studies are based on an established methodology, little is known about the evolution and conservation mechanisms underlying such transcriptomes. Understanding the evolutionary mechanism that change transcriptomes over time, however, might give us a new perspective on how diseases emerge in the first place or how morphological changes are triggered by changes of developmental transcriptomes.</p>
<p>Evolutionary transcriptomics aims to capture and quantify the evolutionary conservation of genes that contribute to the transcriptome during a specific stage of the biological process of interest. The resulting temporal conservation pattern then enables to detect stages of development or other biological processes that are evolutionarily conserved (<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btx835/4772684" class="external-link">Drost et al., 2018</a>). This quantification on the highest level is achieved through transcriptome indices (e.g. <a href="https://drostlab.github.io/myTAI/articles/Introduction.html#transcriptome-age-index" class="external-link">Transcriptome Age Index or Transcriptome Divergence Index</a>) which aim to quantify the average evolutionary age <a href="https://www.biorxiv.org/content/10.1101/2022.07.07.498977v1" class="external-link">Barrera-Redondo et al., 2022</a> or sequence conservation <a href="https://academic.oup.com/mbe/article/32/5/1221/1125964" class="external-link">Drost et al., 2015</a> of genes that contribute to the transcriptome at a particular stage. In general, evolutionary transcriptomics can be used as a method to quantify the evolutionary conservation of transcriptomes at particular developmental stages and to investigate how transcriptomes underlying biological processes are constrained or channeled due to events in evolutionary history (Dollo’s law) (<a href="https://www.sciencedirect.com/science/article/pii/S0959437X16302040" class="external-link">Drost et al., 2017</a>).</p>
<p><strong>Please note, since myTAI relies on gene age inference and there has been an extensive debate about the best approaches for gene age inference in the last years, please follow my <a href="https://drostlab.github.io/myTAI/articles/Phylostratigraphy.html" class="external-link">updated discussion about the gene age inference literature</a>. With <a href="https://github.com/josuebarrera/GenEra" class="external-link">GenEra</a>, we addressed all previously raised issues and we encourage users to run <code>GenEra</code> when aiming to infer gene ages for further <code>myTAI</code> analyses.</strong></p>
</div>
</div>
<div class="section level2">
<h2 id="install-developer-version">Install Developer Version<a class="anchor" aria-label="anchor" href="#install-developer-version"></a>
</h2>
<p>Some bug fixes or new functionality will not be available on CRAN yet, but in the developer version here on GitHub. To download and install the most recent version of <code>myTAI</code> run:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># install developer version of myTAI</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"drostlab/myTAI"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="news">NEWS<a class="anchor" aria-label="anchor" href="#news"></a>
</h2>
<p>The current status of the package as well as a detailed history of the functionality of each version of <code>myTAI</code> can be found in the <a href="https://drostlab.github.io/myTAI/news/index.html" class="external-link">NEWS</a> section.</p>
</div>
<div class="section level2">
<h2 id="tutorials">Tutorials<a class="anchor" aria-label="anchor" href="#tutorials"></a>
</h2>
<p>The following tutorials will provide use cases and detailed explanations of how to quantify transcriptome conservation with <code>myTAI</code> and how to interpret the results generated with this software tool.</p>
<ul>
<li><a href="https://drostlab.github.io/myTAI/articles/Introduction.html" class="external-link">Introduction to the myTAI Package</a></li>
<li><a href="https://drostlab.github.io/myTAI/articles/Intermediate.html" class="external-link">Intermediate Concepts of Phylotranscriptomics</a></li>
<li><a href="https://drostlab.github.io/myTAI/articles/Advanced.html" class="external-link">Advanced Topics of Phylotranscriptomics</a></li>
<li><a href="https://drostlab.github.io/myTAI/articles/Enrichment.html" class="external-link">Perform Age Enrichment Analyses</a></li>
<li><a href="https://drostlab.github.io/myTAI/articles/Expression.html" class="external-link">Gene Expression Analysis with myTAI</a></li>
<li><a href="https://drostlab.github.io/myTAI/articles/Taxonomy.html" class="external-link">Taxonomic Information Retrieval</a></li>
<li><a href="https://drostlab.github.io/myTAI/articles/Phylostratigraphy.html" class="external-link">Discussing Genomic Phylostratigraphy</a></li>
</ul>
</div>
<div class="section level2">
<h2 id="example">Example<a class="anchor" aria-label="anchor" href="#example"></a>
</h2>
<div class="section level4">
<h4 id="load-example-data">Load example data<a class="anchor" aria-label="anchor" href="#load-example-data"></a>
</h4>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/drostlab/myTAI" class="external-link">myTAI</a></span><span class="op">)</span></span>
<span><span class="co"># example dataset covering 7 stages of A thaliana embryo development</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"PhyloExpressionSetExample"</span><span class="op">)</span></span>
<span><span class="co"># transform absolute expression levels to log2 expression levels</span></span>
<span><span class="va">ExprExample</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/tf.html">tf</a></span><span class="op">(</span><span class="va">PhyloExpressionSetExample</span>, <span class="va">log2</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level4">
<h4 id="quantify-transcriptome-conservation-using-tai">Quantify transcriptome conservation using TAI<a class="anchor" aria-label="anchor" href="#quantify-transcriptome-conservation-using-tai"></a>
</h4>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># visualize global Transcriptome Age Index pattern</span></span>
<span><span class="fu"><a href="reference/PlotSignature.html">PlotSignature</a></span><span class="op">(</span><span class="va">ExprExample</span><span class="op">)</span></span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-10-1.png"></p>
</div>
<div class="section level4">
<h4 id="quantify-expression-level-distributions-for-each-gene-age-category">Quantify expression level distributions for each gene age category<a class="anchor" aria-label="anchor" href="#quantify-expression-level-distributions-for-each-gene-age-category"></a>
</h4>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># plot expression level distributions for each age (=PS) category </span></span>
<span><span class="co"># and each developmental stage </span></span>
<span><span class="fu"><a href="reference/PlotCategoryExpr.html">PlotCategoryExpr</a></span><span class="op">(</span><span class="va">ExprExample</span>, <span class="st">"PS"</span><span class="op">)</span></span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-11-1.png"></p>
</div>
<div class="section level4">
<h4 id="quantify-mean-expression-of-individual-gene-age-categories">Quantify mean expression of individual gene age categories<a class="anchor" aria-label="anchor" href="#quantify-mean-expression-of-individual-gene-age-categories"></a>
</h4>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># plot mean expression of each age category seperated by old (PS1-3)</span></span>
<span><span class="co"># versus young (PS4-12) genes</span></span>
<span><span class="fu"><a href="reference/PlotMeans.html">PlotMeans</a></span><span class="op">(</span><span class="va">ExprExample</span>, Groups <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="fl">1</span><span class="op">:</span><span class="fl">3</span>, <span class="fl">4</span><span class="op">:</span><span class="fl">12</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-12-1.png"></p>
</div>
<div class="section level4">
<h4 id="quantify-relative-mean-expression-of-each-age-category-seperated-by-old-versus-young-genes">Quantify relative mean expression of each age category seperated by old versus young genes<a class="anchor" aria-label="anchor" href="#quantify-relative-mean-expression-of-each-age-category-seperated-by-old-versus-young-genes"></a>
</h4>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># plot relative mean expression of each age category seperated by old (PS1-3)</span></span>
<span><span class="co"># versus young (PS4-12) genes</span></span>
<span><span class="fu"><a href="reference/PlotRE.html">PlotRE</a></span><span class="op">(</span><span class="va">ExprExample</span>, Groups <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="fl">1</span><span class="op">:</span><span class="fl">3</span>, <span class="fl">4</span><span class="op">:</span><span class="fl">12</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-13-1.png"></p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># plot the significant differences between gene expression distributions </span></span>
<span><span class="co"># of old (=group1) versus young (=group2) genes</span></span>
<span><span class="fu"><a href="reference/PlotGroupDiffs.html">PlotGroupDiffs</a></span><span class="op">(</span>ExpressionSet <span class="op">=</span> <span class="va">ExprExample</span>,</span>
<span>               Groups        <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>group_1 <span class="op">=</span> <span class="fl">1</span><span class="op">:</span><span class="fl">3</span>, group_2 <span class="op">=</span> <span class="fl">4</span><span class="op">:</span><span class="fl">12</span><span class="op">)</span>,</span>
<span>               legendName    <span class="op">=</span> <span class="st">"PS"</span>,</span>
<span>               plot.type     <span class="op">=</span> <span class="st">"boxplot"</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="getting-started-with-mytai">Getting started with <code>myTAI</code>
<a class="anchor" aria-label="anchor" href="#getting-started-with-mytai"></a>
</h2>
<p>Users can also read the tutorials within (<a href="https://rstudio.com/" class="external-link">RStudio</a>) :</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># source the myTAI package</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/drostlab/myTAI" class="external-link">myTAI</a></span><span class="op">)</span></span>
<span></span>
<span><span class="co"># look for all tutorials (vignettes) available in the myTAI package</span></span>
<span><span class="co"># this will open your web browser</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/browseVignettes.html" class="external-link">browseVignettes</a></span><span class="op">(</span><span class="st">"myTAI"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># or as single tutorials</span></span>
<span></span>
<span><span class="co"># open tutorial: Introduction to Phylotranscriptomics and myTAI</span></span>
<span> <span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"Introduction"</span>, package <span class="op">=</span> <span class="st">"myTAI"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># open tutorial: Intermediate Concepts of Phylotranscriptomics</span></span>
<span> <span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"Intermediate"</span>, package <span class="op">=</span> <span class="st">"myTAI"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># open tutorial: Advanced Concepts of Phylotranscriptomics</span></span>
<span> <span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"Advanced"</span>, package <span class="op">=</span> <span class="st">"myTAI"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># open tutorial: Age Enrichment Analyses</span></span>
<span> <span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"Enrichment"</span>, package <span class="op">=</span> <span class="st">"myTAI"</span><span class="op">)</span></span>
<span> </span>
<span><span class="co"># open tutorial: Gene Expression Analysis with myTAI</span></span>
<span> <span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"Expression"</span>, package <span class="op">=</span> <span class="st">"myTAI"</span><span class="op">)</span></span>
<span> </span>
<span> <span class="co"># open tutorial: Taxonomic Information Retrieval with myTAI</span></span>
<span> <span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"Taxonomy"</span>, package <span class="op">=</span> <span class="st">"myTAI"</span><span class="op">)</span></span>
<span> </span>
<span> <span class="co"># open tutorial: Discussing Genomic Phylostratigraphy</span></span>
<span> <span class="fu"><a href="https://rdrr.io/r/utils/vignette.html" class="external-link">vignette</a></span><span class="op">(</span><span class="st">"Phylostratigraphy"</span>, package <span class="op">=</span> <span class="st">"myTAI"</span><span class="op">)</span></span></code></pre></div>
<p>In the <code>myTAI</code> framework users can find:</p>
<div class="section level4">
<h4 id="phylotranscriptomics-measures">Phylotranscriptomics Measures:<a class="anchor" aria-label="anchor" href="#phylotranscriptomics-measures"></a>
</h4>
<ul>
<li>
<code><a href="reference/TAI.html">TAI()</a></code> : Function to compute the Transcriptome Age Index (TAI)</li>
<li>
<code><a href="reference/TDI.html">TDI()</a></code> : Function to compute the Transcriptome Divergence Index (TDI)</li>
<li>
<code><a href="reference/TPI.html">TPI()</a></code> : Function to compute the Transcriptome Polymorphism Index (TPI)</li>
<li>
<code><a href="reference/REMatrix.html">REMatrix()</a></code> : Function to compute the relative expression profiles of all phylostrata or divergence-strata</li>
<li>
<code><a href="reference/RE.html">RE()</a></code> : Function to transform mean expression levels to relative expression levels</li>
<li>
<code><a href="reference/pTAI.html">pTAI()</a></code> : Compute the Phylostratum Contribution to the global TAI</li>
<li>
<code><a href="reference/pTDI.html">pTDI()</a></code> : Compute the Divergence Stratum Contribution to the global TDI</li>
<li>
<code><a href="reference/pMatrix.html">pMatrix()</a></code> : Compute Partial TAI or TDI Values</li>
<li>
<code><a href="reference/pStrata.html">pStrata()</a></code> : Compute Partial Strata Values</li>
</ul>
</div>
<div class="section level4">
<h4 id="visualization-and-analytics-tools">Visualization and Analytics Tools:<a class="anchor" aria-label="anchor" href="#visualization-and-analytics-tools"></a>
</h4>
<ul>
<li>
<code><a href="reference/PlotSignature.html">PlotSignature()</a></code> : Main visualization function to plot evolutionary signatures across transcriptomes</li>
<li>
<code><a href="reference/PlotPattern.html">PlotPattern()</a></code> : Base graphics function to plot evolutionary signatures across transcriptomes</li>
<li>
<code><a href="reference/PlotContribution.html">PlotContribution()</a></code> : Plot Cumulative Transcriptome Index</li>
<li>
<code><a href="reference/PlotCorrelation.html">PlotCorrelation()</a></code> : Function to plot the correlation between phylostratum values and divergence-stratum values</li>
<li>
<code><a href="reference/PlotRE.html">PlotRE()</a></code> : Function to plot the relative expression profiles</li>
<li>
<code><a href="reference/PlotBarRE.html">PlotBarRE()</a></code> : Function to plot the mean relative expression levels of phylostratum or divergence-stratum classes as barplot</li>
<li>
<code><a href="reference/PlotMeans.html">PlotMeans()</a></code> : Function to plot the mean expression profiles of age categories</li>
<li>
<code><a href="reference/PlotMedians.html">PlotMedians()</a></code> : Function to plot the median expression profiles of age categories</li>
<li>
<code><a href="reference/PlotVars.html">PlotVars()</a></code> : Function to plot the expression variance profiles of age categories</li>
<li>
<code><a href="reference/PlotDistribution.html">PlotDistribution()</a></code> : Function to plot the frequency distribution of genes within the corresponding age categories</li>
<li>
<code><a href="reference/PlotCategoryExpr.html">PlotCategoryExpr()</a></code> : Plot the Expression Levels of each Age or Divergence Category as Barplot or Violinplot</li>
<li>
<code><a href="reference/PlotEnrichment.html">PlotEnrichment()</a></code> : Plot the Phylostratum or Divergence Stratum Enrichment of a given Gene Set</li>
<li>
<code><a href="reference/PlotGeneSet.html">PlotGeneSet()</a></code> : Plot the Expression Profiles of a Gene Set</li>
<li>
<code><a href="reference/PlotGroupDiffs.html">PlotGroupDiffs()</a></code> : Plot the significant differences between gene expression distributions of PS or DS groups</li>
<li>
<code><a href="reference/PlotSelectedAgeDistr.html">PlotSelectedAgeDistr()</a></code> : Plot the PS or DS distribution of a selected set of genes</li>
</ul>
</div>
<div class="section level4">
<h4 id="a-statistical-framework-and-test-statistics">A Statistical Framework and Test Statistics:<a class="anchor" aria-label="anchor" href="#a-statistical-framework-and-test-statistics"></a>
</h4>
<ul>
<li>
<code><a href="reference/FlatLineTest.html">FlatLineTest()</a></code> : Function to perform the <strong>Flat Line Test</strong> that quantifies the statistical significance of an observed phylotranscriptomics pattern (significant deviation from a frat line = evolutionary signal)</li>
<li>
<code><a href="reference/ReductiveHourglassTest.html">ReductiveHourglassTest()</a></code> : Function to perform the <strong>Reductive Hourglass Test</strong> that statistically evaluates the existence of a phylotranscriptomic hourglass pattern (hourglass model)</li>
<li>
<code><a href="reference/EarlyConservationTest.html">EarlyConservationTest()</a></code> : Function to perform the <strong>Reductive Early Conservation Test</strong> that statistically evaluates the existence of a monotonically increasing phylotranscriptomic pattern (early conservation model)</li>
<li>
<code><a href="reference/LateConservationTest.html">LateConservationTest()</a></code> : Function to perform the <strong>Reductive Late Conservation Test</strong> that statistically evaluates the existence of a monotonically decreasing phylotranscriptomic pattern (early divergence model)</li>
<li>
<code>ReverseHourglassTest</code>: Function to perform the <strong>Reverse Hourglass Test</strong> that statistically evaluates the existence of a reverse hourglass pattern (low-high-low)</li>
<li>
<code><a href="reference/EnrichmentTest.html">EnrichmentTest()</a></code> : Phylostratum or Divergence Stratum Enrichment of a given Gene Set based on Fisher’s Test</li>
<li>
<code><a href="reference/bootMatrix.html">bootMatrix()</a></code> : Compute a Permutation Matrix for Test Statistics</li>
</ul>
<p>All functions also include visual analytics tools to quantify the goodness of test statistics.</p>
</div>
<div class="section level4">
<h4 id="differential-gene-expression-analysis">Differential Gene Expression Analysis<a class="anchor" aria-label="anchor" href="#differential-gene-expression-analysis"></a>
</h4>
<ul>
<li>
<code><a href="reference/DiffGenes.html">DiffGenes()</a></code> : Implements Popular Methods for Differential Gene Expression Analysis</li>
<li>
<code><a href="reference/CollapseReplicates.html">CollapseReplicates()</a></code> : Combine Replicates in an ExpressionSet</li>
<li>
<code><a href="reference/CombinatorialSignificance.html">CombinatorialSignificance()</a></code> : Compute the Statistical Significance of Each Replicate Combination</li>
<li>
<code><a href="reference/Expressed.html">Expressed()</a></code> : Filter Expression Levels in Gene Expression Matrices (define expressed genes)</li>
<li>
<code><a href="reference/SelectGeneSet.html">SelectGeneSet()</a></code> : Select a Subset of Genes in an ExpressionSet</li>
<li>
<code><a href="reference/PlotReplicateQuality.html">PlotReplicateQuality()</a></code> : Plot the Quality of Biological Replicates</li>
<li>
<code><a href="reference/GroupDiffs.html">GroupDiffs()</a></code> : Quantify the significant differences between gene expression distributions of PS or DS groups</li>
</ul>
</div>
<div class="section level4">
<h4 id="taxonomic-information-retrieval">Taxonomic Information Retrieval<a class="anchor" aria-label="anchor" href="#taxonomic-information-retrieval"></a>
</h4>
<ul>
<li>
<code><a href="reference/taxonomy.html">taxonomy()</a></code> : Retrieve Taxonomic Information for any Organism of Interest</li>
</ul>
</div>
<div class="section level4">
<h4 id="minor-functions-for-better-usibility-and-additional-analyses">Minor Functions for Better Usibility and Additional Analyses<a class="anchor" aria-label="anchor" href="#minor-functions-for-better-usibility-and-additional-analyses"></a>
</h4>
<ul>
<li>
<code><a href="reference/MatchMap.html">MatchMap()</a></code> : Match a Phylostratigraphic Map or Divergence Map with a ExpressionMatrix</li>
<li>
<code><a href="reference/tf.html">tf()</a></code> : Transform Gene Expression Levels</li>
<li>
<code><a href="reference/tfStability.html">tfStability()</a></code> : Perform Permutation Tests Under Different Transformations</li>
<li>
<code><a href="reference/age.apply.html">age.apply()</a></code> : Age Category Specific apply Function</li>
<li>
<code><a href="reference/ecScore.html">ecScore()</a></code> : Compute the Hourglass Score for the EarlyConservationTest</li>
<li>
<code><a href="reference/lcScore.html">lcScore()</a></code> : Compute the Hourglass Score for the LateConservationTest</li>
<li>
<code><a href="reference/geom.mean.html">geom.mean()</a></code> : Geometric Mean</li>
<li>
<code><a href="reference/harm.mean.html">harm.mean()</a></code> : Harmonic Mean</li>
<li>
<code><a href="reference/omitMatrix.html">omitMatrix()</a></code> : Compute TAI or TDI Profiles Omitting a Given Gene</li>
<li>
<code><a href="reference/rhScore.html">rhScore()</a></code> : Compute the Hourglass Score for the Reductive Hourglass Test</li>
<li>
<code><a href="reference/reversehourglassScore.html">reversehourglassScore()</a></code>: Compute the Reverse Hourglass Score for the Reverse Hourglass Test</li>
</ul>
</div>
</div>
<div class="section level2">
<h2 id="studies-that-successfully-used-mytai-to-quantify-transcriptome-conservation">Studies that successfully used <code>myTAI</code> to quantify transcriptome conservation:<a class="anchor" aria-label="anchor" href="#studies-that-successfully-used-mytai-to-quantify-transcriptome-conservation"></a>
</h2>
<blockquote>
<ul>
<li><p><strong>Evolutionary transcriptomics of metazoan biphasic life cycle supports a single intercalation origin of metazoan larvae</strong> J Wang, L Zhang, S Lian, Z Qin, X Zhu, X Dai, Z Huang et al. - <strong>Nature Ecology &amp; Evolution</strong>, 2020</p></li>
<li><p><strong>Pervasive convergent evolution and extreme phenotypes define chaperone requirements of protein homeostasis</strong> Y Draceni, S Pechmann - <strong>Proceedings of the National Academy of Sciences</strong>, 2019</p></li>
<li><p><strong>Reconstructing the transcriptional ontogeny of maize and sorghum supports an inverse hourglass model of inflorescence development</strong> S Leiboff, S Hake - <strong>Current Biology</strong>, 2019</p></li>
<li><p><strong>A unicellular relative of animals generates a layer of polarized cells by actomyosin-dependent cellularization</strong> O Dudin, A Ondracka, X Grau-Bové, AAB Haraldsen et al. - <strong>eLife</strong>, 2019</p></li>
<li><p><strong>Evidence for active maintenance of phylotranscriptomic hourglass patterns in animal and plant embryogenesis</strong> HG Drost, A Gabel, I Grosse, M Quint - <strong>Molecular biology and evolution</strong>, 2015</p></li>
<li><p><strong>Gene Expression Does Not Support the Developmental Hourglass Model in Three Animals with Spiralian Development</strong> L Wu, KE Ferger, JD Lambert - <strong>Molecular biology and evolution</strong>, 2019</p></li>
<li><p><strong>Transcriptome age of individual cell types in Caenorhabditis elegans</strong> F Ma, C Zheng - <strong>Proceedings of the National Academy of Sciences</strong>, 2023</p></li>
<li><p><strong>The Transcriptional Landscape of Polyploid Wheats and their Diploid Ancestors during Embryogenesis and Grain Development</strong> D Xiang, TD Quilichini, Z Liu, P Gao, Y Pan et al. - <strong>The Plant Cell</strong>, 2019</p></li>
<li><p><strong>Embryo-Like Features in Developing Bacillus subtilis Biofilms</strong> M Futo, L Opašić, S Koska, N Čorak, T Široki, V Ravikumar, A Thorsell, M Lenuzzi, D Kifer, M Domazet-Lošo, K Vlahoviček, I Mijakovic, T Domazet-Lošo - <strong>Molecular Biology and Evolution</strong>, 2021</p></li>
<li><p><strong>New Genes Interacted With Recent Whole-Genome Duplicates in the Fast Stem Growth of Bamboos</strong> G Jin, P-F Ma, X Wu, L Gu, M Long, C Zhang, DZ Li - <strong>Molecular Biology and Evolution</strong>, 2021</p></li>
<li><p><strong>Developmental constraints on genome evolution in four bilaterian model species</strong> J Liu, M Robinson-Rechavi - <strong>Genome biology and evolution</strong>, 2018</p></li>
<li><p><strong>Mapping selection within Drosophila melanogaster embryo’s anatomy</strong> I Salvador-Martínez et al. - <strong>Molecular biology and evolution</strong>, 2017</p></li>
<li><p><strong>Distribution and diversity of enzymes for polysaccharide degradation in fungi</strong> R Berlemont - <strong>Scientific reports</strong>, 2017</p></li>
<li><p><strong>The origins and evolutionary history of human non-coding RNA regulatory networks</strong> M Sherafatian, SJ Mowla - <strong>Journal of bioinformatics and computational biology</strong>, 2017</p></li>
<li><p><strong>Elucidating the endogenous synovial fluid proteome and peptidome of inflammatory arthritis using label-free mass spectrometry</strong> SM Mahendran, EC Keystone, RJ Krawetz et al. - <strong>Clinical proteomics</strong>, 2019</p></li>
<li><p><strong>Phylostratr: a framework for phylostratigraphy</strong> Z Arendsee, J Li, U Singh, A Seetharam et al. - <strong>Bioinformatics</strong>, 2019</p></li>
<li><p><strong>Pervasive convergent evolution and extreme phenotypes define chaperone requirements of protein homeostasis</strong> Y Draceni, S Pechmann - <strong>BioRxiv</strong>, 2019</p></li>
<li><p><strong>Environmental DNA reveals landscape mosaic of wetland plant communities</strong> ME Shackleton, GN Rees, G Watson et al. - <strong>Global Ecology and Conservation</strong>, 2019</p></li>
<li><p><strong>Algorithms for synteny-based phylostratigraphy and gene origin classification</strong> Z Arendsee - 2019</p></li>
<li><p><strong>High expression of new genes in trochophore enlightening the ontogeny and evolution of trochozoans</strong> F Xu, T Domazet-Lošo, D Fan, TL Dunwell, L Li et al. - <strong>Scientific reports</strong>, 2016</p></li>
<li><p><strong>Oxytocin receptor expression patterns in the human brain across development</strong> J Rokicki, T Kaufmann, A-MG de Lange, D van der Meer, S Bahrami, AM Sartorius, UK Haukvik, NE Steen, E Schwarz, DJ Stein, T Nærland, OA Andreassen, LT Westlye, DS Quintana - <strong>Neuropsychopharmacol</strong>, 2022</p></li>
<li><p><strong>The Phylotranscriptomic Hourglass Pattern in Fungi: An Updated Model</strong> Y Xie, HS Kwan, PL Chan, WJ Wu, J Chiou, J Chang <strong>BioRxiv</strong>, 2022</p></li>
<li><p><strong>An ancient split of germline and somatic stem cell lineages in Hydra</strong> C Nishimiya-Fujisawa, H Petersen, TC-T Koubková-Yu, C Noda, S Shigenobu, J Bageritz, T Fujisawa, O Simakov, S Kobayashi, TW Holstein - <strong>BioRxiv</strong>, 2023</p></li>
</ul>
</blockquote>
</div>
<div class="section level2">
<h2 id="discussions-and-bug-reports">Discussions and Bug Reports<a class="anchor" aria-label="anchor" href="#discussions-and-bug-reports"></a>
</h2>
<p>I would be very happy to learn more about potential improvements of the concepts and functions provided in this package.</p>
<p>Furthermore, in case you find some bugs or need additional (more flexible) functionality of parts of this package, please let me know:</p>
<p><a href="https://github.com/drostlab/myTAI/issues" class="external-link uri">https://github.com/drostlab/myTAI/issues</a></p>
</div>
<div class="section level2">
<h2 id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<p>Domazet-Lošo T. and Tautz D. <strong>A phylogenetically based transcriptome age index mirrors ontogenetic divergence patterns</strong>. <em>Nature</em> (2010) 468: 815-8.</p>
<p>Quint M, Drost HG, et al. <strong>A transcriptomic hourglass in plant embryogenesis</strong>. <em>Nature</em> (2012) 490: 98-101.</p>
<p>Drost HG, Gabel A, Grosse I, Quint M. <strong>Evidence for Active Maintenance of Phylotranscriptomic Hourglass Patterns in Animal and Plant Embryogenesis</strong>. <em>Mol. Biol. Evol.</em> (2015) 32 (5): 1221-1231.</p>
<p>Drost HG, Bellstädt J, Ó’Maoiléidigh DS, Silva AT, Gabel A, Weinholdt C, Ryan PT, Dekkers BJW, Bentsink L, Hilhorst H, Ligterink W, Wellmer F, Grosse I, and Quint M. <strong>Post-embryonic hourglass patterns mark ontogenetic transitions in plant development</strong>. <em>Mol. Biol. Evol.</em> (2016) <a href="https://academic.oup.com/mbe/article/33/5/1158/2580081" class="external-link">doi:10.1093/molbev/msw039</a></p>
</div>
<div class="section level2">
<h2 id="acknowledgement">Acknowledgement<a class="anchor" aria-label="anchor" href="#acknowledgement"></a>
</h2>
<p>I would like to thank several individuals for making this project possible.</p>
<p>First I would like to thank Ivo Grosse and Marcel Quint for providing me a place and the environment to be able to work on fascinating topics of Evo-Devo research and for the fruitful discussions that led to projects like this one.</p>
<p>Furthermore, I would like to thank Alexander Gabel and Jan Grau for valuable discussions on how to improve some methodological concepts of some analyses present in this package.</p>
<p>I would also like to thank my past Master Students: Sarah Scharfenberg, Anne Hoffmann, and Sebastian Wussow who worked intensively with this package and helped me to improve the usability and logic of the package environment.</p>
</div>
</div>

  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2 data-toc-skip>Links</h2>
<ul class="list-unstyled">
<li><a href="https://cloud.r-project.org/package=myTAI" class="external-link">View on CRAN</a></li>
<li><a href="https://github.com/drostlab/myTAI/" class="external-link">Browse source code</a></li>
<li><a href="https://github.com/drostlab/myTAI/issues" class="external-link">Report a bug</a></li>
</ul>
</div>

<div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li><a href="https://www.r-project.org/Licenses/GPL-3" class="external-link">GPL-3</a></li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing myTAI</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>Hajk-Georg Drost <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0002-1567-306X" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Kristian K Ullrich <br><small class="roles"> Author, contributor </small> <a href="https://orcid.org/0000-0003-4308-9626" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Jaruwatana Sodai Lotharukpong <br><small class="roles"> Author, contributor </small> <a href="https://orcid.org/0000-0002-3475-0980" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Nikola Kalábová <br><small class="roles"> Author, contributor </small> <a href="https://orcid.org/0009-0007-6384-8809" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>



  </div>
</div>


      <footer><div class="copyright">
  <p></p>
<p>Developed by Hajk-Georg Drost, Kristian K Ullrich, Jaruwatana Sodai Lotharukpong, Nikola Kalábová.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
